Include/cpython/pyerrors.h

```diff
@@ -82,6 +82,13 @@ typedef struct {
     PyObject *name;
 } PyAttributeErrorObject;
 
+typedef struct {
+    PyException_HEAD
+    PyObject *iterable;
+    // TODO: argcnt is probably not the right name for this variable
+    PyObject *argcnt;
```

Comment:
How about `count` for the expected count? Or `expected_count`? You could also have an `actual_count` member set to None so that users could make their own UnpackError. A bit like `OSError` where you have filename and filename2.

Reply:
`actual_count` seems unnecessary because I'd like `iterable` to always be non-null.